app/src/main/java/me/saket/dank/ui/user/UserSessionRepository.java

```diff
@@ -45,6 +61,39 @@ public Completable logout() {
         .andThen(Completable.fromAction(() -> removeLoggedInUsername()));
   }
 
+  public SortingAndTimePeriod getSortingAndTimePeriodForSub(String subredditName) {
```

Comment:
We can avoid using `get` prefix for getters. Just `sortingAndTimePeriodForSub()` should be sufficient.

Reply:
Maybe `lastUsedSortingAndTimePeriod()` would be a better name